crates/edr_rpc_client/src/client.rs

```diff
@@ -916,6 +894,28 @@ mod tests {
             }
         }
 
+        #[tokio::test]
+        async fn call_bad_api_key() {
```

Comment:
Moved this test under the `test-remote` flag because it requires an internet connection.